docs/developer-guide/ksqldb-reference/create-table.md

```diff
@@ -29,13 +29,21 @@ information, see [Parallelization](/operate-and-deploy/performance-guidelines/#p
 
 ### PRIMARY KEY
 
-A ksqlDB TABLE works much like tables in other SQL systems. A table has zero or more rows. Each
-row is identified by its `PRIMARY KEY`. A row's `PRIMARY KEY` can not be `NULL`. A message in the
-underlying Kafka topic with the same key as an existing row will _replace_ the existing row in the
-table, or _delete_ the row if the message's value is `NULL`, i.e. a _tombstone_, as long as it has
-a later timestamp / `ROWTIME`.
+A ksqlDB TABLE works much like tables in other SQL systems. A table has zero or
+more rows. Each row is identified by its `PRIMARY KEY`. A row's `PRIMARY KEY`
+can't be `NULL`.
 
-This situation is handled differently by [ksqlDB STREAM](../create-stream), as shown in the following table.
+If an incoming message in the underlying {{ site.ak }} topic has the same key
+as an existing row, it _replaces_ the existing row in the table. But if the
+message's value is `NULL`, it _deletes_ the row, as long as it has a later

```

Comment:
> But if the message's value is `NULL`

This is actually miss-leading, because there is a difference between the Kafka key/value model and ksqlDB data model. A `<key,null>` is a tombstone, but a `<key, {}>` is not, but for both cases the SQL columns are `NULL`.

We should at least use `null` instead of `NULL` (as `NULL` is a logical SQL term, while `null` is a Java term).